### ISSUE TITLE
The difference in rendering between Dioxus and native HTML + CSS.

### ISSUE THREAD

**Issue by faunel** [DioxusLabs/dioxus#1783]
**Problem**

When I run the "calculator" example, it is not displaying correctly.
But when I manually created the same HTML and copied the CSS (assets/calculator.css), everything is displayed correctly.

In the screenshot on the left is Dioxus, and on the right is HTML + CSS.

**Steps To Reproduce**

```
cargo run --example calculator
```

**Expected behavior**

I expect there to be no difference between Dioxus and native HTML + CSS.

**Screenshots**

<img width="646" alt="image" src="https://github.com/DioxusLabs/dioxus/assets/18446512/abeeb0b0-5b37-459c-a764-6b758c4f1e66">

**Environment:**
 - Dioxus version: 0.4.3
 - Rust version: 1.74.1
 - OS info: MacOS M1
 - App platform: desktop

**Comment by ealmloff** [comment]
Different browsers have different default style sheets (you look at the differences [here](https://browserdefaultstyles.com)). Dioxus uses your system webview to render your desktop app. If you open it in a different browser, it may display slightly differently. You can use something like [css reset](https://meyerweb.com/eric/tools/css/reset/) or tailwind to remove 
inconsistencies between browsers default styles

**Comment by faunel** [comment]
I tried it in different browsers - Chrome, Firefox, Safari, and it displays the same (correctly) in all of them. I also tried using reset.css, but unfortunately, it didn't help.

It seems like I've figured out the issue. In the desktop version of the program, the height needs to take into account the top window frame (where the calculator label is). To accommodate this, I had to slightly reduce the height of the 'display' element. In my case, it's 30px. These pixels can be removed either from the 'display' element or from the keys. I'm not sure if this is a bug or a feature.

<img width="644" alt="image" src="https://github.com/DioxusLabs/dioxus/assets/18446512/2ef1b8a2-97e0-410b-abce-5cdba0be1cb8">
